src/Microsoft.PowerShell.Commands.Management/commands/management/SetClipboardCommand.cs

```diff
@@ -299,21 +299,21 @@ private void CopyFilesToClipboard(List<string> fileList, bool append, bool isLit
             }
         }
 
-        /// <summary>      
-        /// Generate HTML fragment data string with  header that is required for the clipboard.      
-        /// </summary>      
-        /// <param name="html">the  html to generate for</param>      
-        /// <returns>the resulted  string</returns>      
+        /// <summary>
+        /// Generate HTML fragment data string with  header that is required for the clipboard.
+        /// </summary>
+        /// <param name="html">the  html to generate for</param>
+        /// <returns>the resulted  string</returns>
         private static string GetHtmlDataString(string html)
         {
-            // The string contains index references to  other spots in the string, so we need placeholders so we can compute the  offsets.   
-            // The  "<<<<<<<<1,<<<<<<<<2, etc" strings are just placeholders.  We'll back-patch them actual values within the header location afterwards.    
-            const string Header = @"Version:0.9      
```

Comment:
You're right, there is some small risk here (not this change, it looks fine after looking at how the string is used.)

I'm fine with accepting the risk, this change will simplify reviews in the future when folks use editors that trim trailing whitespace.